goaviatrix/security_domain.go

```diff
@@ -207,6 +222,31 @@ func (c *Client) DisableIntraDomainInspection(ctx context.Context, intraDomainIn
 	return c.PostAPIContext(ctx, params["action"], params, BasicCheck)
 }
 
+func (c *Client) GetAllSecurityDomains() ([]NetworkDomainDetails, error) {
```

Comment:
I think we agreed to name all new methods with Network Domain, so this would be `GetAllNetworkDomains`

Reply:
Have fixed